root/static/scripts/common/components/TagEditor.js

```diff
@@ -365,6 +368,26 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
     input.value = '';
   }
 
+  handleChange() {
+    const input = this.tagsInput;
+    if (!input) {
+      return;
+    }
+
+    /*
+     * MBS-9862: jQuery UI disables the browser's builtin autocomplete
+     * history in a non-configurable way, but we want it to show here
+     * if the user hasn't typed anything yet, so flip it back on.
+     * Turn it off again if something has been typed (activating
+     * the jQuery autocomplete).
+     */
+    if (input.value === '') {
+      input.setAttribute('autocomplete', 'on');
+    } else {
+      input.setAttribute('autocomplete', 'off');
+    }
```

Comment:
This minifies better and seems equally easy to understand:

```suggestion
    /*
     * MBS-9862: jQuery UI disables the browser's builtin autocomplete
     * history in a non-configurable way, but we want it to show here
     * if the user hasn't typed anything yet, so flip it back on.
     * Turn it off again if something has been typed (activating
     * the jQuery autocomplete).
     */
    this.tagsInput?.setAttribute('autocomplete', input.value ? 'off' : 'on');
```